ios/Mappers.swift

```diff
@@ -658,12 +658,19 @@ class Mappers {
     }
 
     class func mapFromPaymentMethod(_ paymentMethod: PaymentMethod) -> NSDictionary {
+        var cardPresentMapped: NSDictionary?
+        if let cardPresent = paymentMethod.cardPresent{
+            cardPresentMapped = mapFromCardPresent(cardPresent)
+        }
+        var interacPresentMapped: NSDictionary?
+        if let interacPresent = paymentMethod.interacPresent{
+            interacPresentMapped = mapFromCardPresent(interacPresent)
+        }
```

Comment:
thanks for fixing 🙏 

non-blocking suggestion: could also update mapFromCardPresent to accept a nullable `CardPresentDetails` and then can re-inline these.